code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -239,37 +239,6 @@
 	update_icon()
 	return
 
-/obj/item/gun/ballistic/automatic/smg/greasegun/worn
```

Comment:
why are we removing this

Reply:
How did that get removed? Let me fix that.

Reply:
> How did that get removed? Let me fix that.

its likely the cause of your error lol, we tried to TM this and had a whole mess pulling it back out because it was a path not found

Reply:
Should be good now.